content/en/seachat/getting-started/02-quick-start.md

```diff
@@ -0,0 +1,143 @@
+
+> :books: **Agent and Live Agent**
+>
+>You will see the term, **Agent**, used throughout this tutorial. An agent is essentially an AI assistance with a custom knowledge base, we use the term, **Agent** to separate its capabilities from a traditional chatbot. Whereas a **Live Agent** refers to a human agent who interacts with users in real-time.
+
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem; transition: transform 0.5s ease;" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step2.jpg" alt="">
+<br/>
+
+<br/>
+
+Once you have picked the right name for your agent, move on to **Description** to give your agent some context of its role and functions before you choose its use case and response language. I have picked a very creative name **My Parking Agent** for this tutorial.  I will choose **FAQ** as my use case, since this agent will be used to answer frequently asked questions, and **US English** for the response language.
+<br/>
+
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step2-2.jpg" alt="">
+<br/>
+
+<br/>
+
+> :bulb: **Note** 
+>
+>  The live agent feature will require a human agent in case the customer wishes to be transferred to one. The customer will then have the option to request a human agent during the chat.
+> 
+
+## Step 3:  Add Knowledge
+To fully leverage the capabilities of your SeaChat agent, we need to add knowledge to its knowledge base. SeaChat offers a range of options for uploading content to your agent's knowledge base. Find your agent's knowledge base by navigating to the **Knowledge Base** dashboard under **Agent Configuration** in the sidebar menu. 
+
+Feel free to try out the different ways to upload data to your knowledge base. For the purposes of this tutorial, we'll focus on utilizing the spreadsheet upload method. However, if you're interested in exploring additional data uploading techniques, we invite you to consult the [Add Knowledge]( /en/seachat/seachat-intro/03-add-knowledge/add-knowledge-intro) section within our user manual for comprehensive instructions and tips.
+<br/>
+<img width="80%" style="border-radius: 0.4rem" src="/images/product-updates/seachat/en/tutorial-intro/20240311-tutorial-intro-step3.jpg" alt="">
+<br/>
+
+## Step  4:  Upload Files to Knowledge Base
+Uploading a spreadsheet is as simple as clicking on the upload icon and choosing the desired spreadsheet. Once you see your uploaded files in the preview section below the drag-and-drop section, click on the **Next** button to confirm. 
+Now, ncc_Car_Parks (1) is uploaded, adn we can see each header and its corresponding data in the preview section. My Parking Agent is now ready for the next step.
```

Comment:
do you plan to upload this csv to google drive and let people download it. I think that'd be nice. If you do, make this file clickable and redirects to download